azkaban-common/src/main/java/azkaban/version/VersionSet.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.version;
+
+/**
+ * VersionSet is defined as the set of key-val pairs to uniquely define the execution environment.
+ * This version set will have azkaban version, azkaban configuration version, job type versions, etc.
+ *
+ * Example version set json:
+ * {"azConf":"0.0.5","azkaban":"0.0.2","sparkJobType":"0.1.2"}
+ */
+public class VersionSet {
+    private final String versionSetJsonString;
+    private final String versionSetMd5Hex;
+    private final int versionSetNum;
```

Comment:
Is this id, primary key in a table?

Reply:
versionSetNum will be an autoincrement field.

Reply:
Yup so it's name should be id right?